nrf_modem/include/nrf_socket.h

```diff
@@ -323,6 +326,50 @@ extern "C" {
  *       modem firmware version 2.0.2 and newer.
  */
 #define NRF_SO_IPV6_DELAYED_ADDR_REFRESH 62
+/**
+ * Set a callback to be called when a send operation is done sending data on air and
+ * the data has been acknowledged by the peer, if required by the network protocol, or until the
+ * timeout, given by the @ref NRF_SO_SNDTIMEO socket option, is reached. Valid timeout values are
+ * 1 to 600 seconds.
+ * See @ref nrf_modem_sendcb_t for the callback function type.
+ *
+ * @note The callback is executed in an interrupt context.
+ *	 Take care to offload any processing as appropriate.
+ *
+ * @note This is only supported by the following modem firmware:
+ *       - mfw_nrf9151-ntn
```

Comment:
```suggestion
 * @note This is only supported by the nRF9151 NTN modem firmware.
```
since we just have a single entry.

Reply:
Kept as is to maintain the established format, will update the remaining entries in this file to ensure consistency in a follow-up PR.